docs/asset-manager-framework.md

```diff
@@ -113,19 +113,58 @@ add_action( 'amf/register_providers', function ( ProviderRegistry $provider_regi
 
 For a more complete example using Unsplash see the [AMF Unsplash integration plugin](https://github.com/humanmade/amf-unsplash) for reference.
 
-### Extending Existing Providers
+### Dynamic Image Resizing
 
-Since you have access to each provider instance during registration via the `amf/provider` filter, you can also use it and decorate it:
+AMF provides an interface to enable dynamic image resizing for your provider classes. The Global Media Library uses this to deliver assets via Tachyon out of the box.
+
+To make a provider that can resize assets on the fly it needs to implement the `Resize` interface:
+
+```php
+use UnsplashProvider;
+use AssetManagerFramework\Interfaces\{
+    Resize
+};
```

Comment:
Perfectly valid way to use multiple classes from a namespace. Could be simplified down, I just went with this because there’s going to be more interfaces added in future

Reply:
works for me. presumably this example will expand in the future to include other classes in which case that might be more obvious, I've just never seen this syntax before.

Reply:
iirc we disallowed this in our coding standards 🤔

Reply:
how come? Or just if there's only a single thing referenced? I updated it and the code example above anyway to reflect that.

Reply:
So that it can be alphabetised, iirc